### ISSUE TITLE
Add support for 6-symbol QTH locators

### ISSUE THREAD

**Issue by sp9dev** [JamesP6000/WsprryPi#40]
Hi! Could you add support for transmitting 6-symbol locators? Thanks!

**Comment by MichaelRenner** [comment]
An good idea. Support for type 2 and type 3 messages. Would include "long" call sign, e.g. if you are abroad.

**Comment by ComputerSmiths** [comment]
Yeah, I just brought my TAPR Hat to the Turks and Caicos, and discovered VP5/N1JBJ doesn't work.  Also, my NTP server's gpsd gives me an 8-character grid square, so if I'm going to be doing two transmissions, I might as well narrow down exactly where the only  WSPR transmitter in the TCI is  8*)
